.github/workflows/continuous-integration-workflow.yml

```diff
@@ -30,32 +30,26 @@ jobs:
 
         # We want to make sure proper formatting of the notebooks.
         flake8-nb; [ $? -eq 0 ] || exit 1
-        black-nb .; [ $? -eq 0 ] || exit 1
+        black-nb --check .; [ $? -eq 0 ] || exit 1
 
         # We want to pull the submodules.
         # This can be removed once temfpy is live on conda.
```

Comment:
This can go, right?

Reply:
Er, no, do we not want to check black-nb? I actually copied this from your last fix branch…